src/app/zap-templates/zcl/data-model/chip/application-launcher-cluster.xml

```diff
@@ -46,7 +46,7 @@ limitations under the License.
 
     <command source="server" code="0x03" name="LauncherResponse" optional="false">
       <description>This command SHALL be generated in response to LaunchApp commands.</description>
-      <arg name="status" type="StatusEnum"/>
+      <arg name="status" type="StatusEnumApplicationLauncher"/>
```

Comment:
Why this, and other non-spec-matching changes to names?

Reply:
Commented above on this : So we have multiple definitions of StatusEnum which are of type enum8 defined across our xml files. Loading the data type schema becomes a lot less complicated this way because it is easier to refer to the right types from other parts of the schema. I wanted to keep the data type schema mutually exclusive from cluster references as much as possible because that will help with scaling up the data types and zcl schema easily in the future. So I changed the references to these types with a reference to the Cluster they belong to in their naming.